source/_integrations/androidtv.markdown

```diff
@@ -179,14 +60,22 @@ When connecting to your device for the first time, a dialog will appear on your
 
 ### 1. Python ADB Implementation
 
-The default approach is to connect to your device using the `adb-shell` Python package. As of Home Assistant 0.101, if a key is needed for authentication and it is not provided by the `adbkey` configuration option, then Home Assistant will generate a key for you.
+The default approach is to connect to your device using the `adb-shell` Python package. As of Home Assistant 0.101, if a key is needed for authentication and it is not provided by the `ADB Key` setup option, then Home Assistant will generate a key for you.
+
+<div class='note'>
+To be able to provide `ADB Key` on integration setup, you need to enable [advanced mode](https://www.home-assistant.io/blog/2019/07/17/release-96/#advanced-mode).
+</div>
```

Comment:
Adjustment to the link and an empty line before after language switching to help the Markdown parse to switch HTML/Markdown context.

```suggestion
<div class='note'>

To be able to provide `ADB Key` on integration setup, you need to enable [advanced mode](/blog/2019/07/17/release-96/#advanced-mode).

</div>
```